doc/git-subrepo.swim

```diff
@@ -216,8 +216,16 @@ the same arguments. Keep reading…
   discouraged. Only use this option if you fully understand it. (The `--force`
   option will NOT check for a proper merge. ANY branch will be force pushed!)
 
+  The `--tag` option will apply a given (non-annotated) tag to the specified
+  subrepos' upstream HEAD, provided that they were staged for changes, leaving
+  the parent repository unchanged.
+
+  The `--tag-refresh` will apply the given tag (with `--tag`) to all specified
```

Comment:
The way I read this, it would be the equivalent to `git subrepo push --tag --all --force`. Am I correct? If so I prefer calling out all the parameters to --tag-refresh.